docker/docker/Dockerfile.linux.amd64

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/kaniko-project/executor:v1.6.0
+FROM gcr.io/kaniko-project/executor:v1.8.0
```

Comment:
@aman-harness Can you test whether updated kaniko works for gcr, ecr & dockerhub? Last time there was an issue in updated kaniko version due to which we have to revert the change.

Reply:
[](url
<img width="1559" alt="Screenshot 2022-03-11 at 12 22 16 AM" src="https://user-images.githubusercontent.com/10278482/157734551-8fd818d3-24f5-4bfe-8ca8-421d2b151154.png">
)
I ahve tested it for all 3 of them and it was working fine